src/main/java/racingcar/view/InputView.java

```diff
@@ -1,17 +1,23 @@
 package racingcar.view;
 
+import racingcar.entity.Name;
+
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
+import java.util.stream.Collectors;
 
 public class InputView {
     public static final Scanner scanner = new Scanner(System.in);
     public static final String COMMA_SEPARATOR = ",";
 
-    public String[] requestCarNames() {
+    public static List<Name> requestCarNames() {
         System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
-        return scanner.nextLine().split(COMMA_SEPARATOR);
+        String[] split = scanner.nextLine().split(COMMA_SEPARATOR);
+        return Arrays.stream(split).map(name -> new Name(name)).collect(Collectors.toList());
```

Comment:
스트림 👍 
줄바꿈을 해주시면 가독성을 향상시킬 수 있습니다ㅎㅎ
```suggestion
        return Arrays.stream(split)
        .map(name -> new Name(name))
        .collect(Collectors.toList());
```